pyanaconda/packaging/__init__.py

```diff
@@ -1080,6 +1080,32 @@ def _setupInstallDevice(self, storage, checkmount):
             url = method.url
             mirrorlist = method.mirrorlist
             sslverify = not (method.noverifyssl or flags.noverifyssl)
+        elif method.method == "hmc":
+            # Check if /dev/hmcdrv is already mounted.
```

Comment:
Could you please move this to its own method? This whole method is big as hell already and I don't think we want to have it even bigger.